dt-users/user-management.php

```diff
@@ -15,7 +15,7 @@ public static function instance() {
     public function __construct() {
         $url_path = dt_get_url_path();
         if ( self::has_permission() || self::non_admins_can_make_users() ) {
-            if ( strpos( $url_path, 'user-management/user' ) !== false || ( strpos( $url_path, 'user-management/add-user' ) !== false && ( current_user_can( "create_users" ) || self::non_admins_can_make_users() ) ) ){
+            if ( strpos( $url_path, 'user-management/user' ) !== false || strpos( $url_path, 'user-management/hover-map' ) !== false || ( strpos( $url_path, 'user-management/add-user' ) !== false && ( current_user_can( "create_users" ) || self::non_admins_can_make_users() ) ) ){
```

Comment:
@kodinkat Lets find a better strategy for when we add more user management pages.

Reply:
@corsacca As `Hover Map` template is shared across metrics and user-management views, decided to shift away from class name, but instead; based conditional logic on `url_path` structure...